src/store.js

```diff
@@ -0,0 +1,79 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import axios from '@/axiosInstance';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    status: '',
+    token: localStorage.getItem('token') || '',
+    user: {},
+  },
+  mutations: {
+    auth_request(state) {
+      state.status = 'loading';
+    },
+    auth_success(state, token, user) {
+      state.status = 'success';
+      state.token = token;
+      state.user = user;
+    },
+    auth_error(state) {
+      state.status = 'error';
+    },
+    logout(state) {
+      state.status = '';
+      state.token = '';
+    },
+  },
+  actions: {
```

Comment:
Waarom zijn alle deze functies in de store? Ze worden maar op één plek gebruikt toch.

Reply:
Ze zijn er neergezet met de gedachte. Dat ze later in het ontwikkelproces nog hergebruikt moeten worden.